loadtest/tests.go

```diff
@@ -42,8 +42,7 @@ type UserEntity struct {
 }
 
 func readTestFile(name string) ([]byte, error) {
-	path, _ := utils.FindDir("testfiles")
-	file, err := os.Open(path + "/" + name)
+	file, err := os.Open(name)
```

Comment:
`utils.FindDir` no longer exists, but we didn't need all that fanciness, so I've just hard-coded the `testfiles` path instead.